modules/drivers/oracle/src/metabase/driver/oracle.clj

```diff
@@ -446,6 +448,13 @@
   [_ entity-name]
   (str/replace entity-name "/" "//"))
 
+(defmethod sql-jdbc.describe-table/get-table-pks :oracle
+  [_driver ^Connection conn _ table]
```

Comment:
```suggestion
  [_driver ^Connection conn _db-name-or-nil table]
```

It's a little easier to read this way. Otherwise you have to go look at the `defmulti` to figure out which parameter we're not using.